hopli/README.md

```diff
@@ -37,290 +24,204 @@ An additional parameter which specifies its prefix can also be passed.
-IDENTITY_PASSWORD=local \
-    hopli identity \
-    --action create \
-    --identity-directory "./test" \
-    --identity-prefix node_ \
-    --number 3
+hopli identity create \
+  --identity-directory "./test" \
+  --identity-prefix nodes_ \
+  --number 2 \
+  --password-path "./test/pwd"
 ```
 
-Read ethereum addresses from identities
+#### Read identities
+Read ethereum addresses from identities. A [path to it](####Identity-directory-or-path) and a [password](####Password) must be provided.
 
 ```
-IDENTITY_PASSWORD=switzerland \
-    hopli identity \
-    --action read \
-    --identity-directory "./test" \
-    --identity-prefix node_
+hopli identity read \
+  --identity-directory "./test" \
+  --identity-prefix node_ \
+  --password-path "./test/pwd"
+```
+
+#### Update identities password
+To update some identites' password. Two [passwords](####Password) must be provided:
+- an old (and valid) password to the identity files. Either provided as cli argument `--password-path` or as an env variable `IDENTITY_PASSWORD`
+- a new password to the identity files. Either provided as cli argument `--new-password-path` or as an env variable `NEW_IDENTITY_PASSWORD`
 
 ```
+hopli identity update \
+  --identity-directory "./test" \
+  --identity-prefix node_ \
+  --password-path "./test/pwd" \
+  --new-password-path "./test/newpwd"
+```
+
+The identities will be modified inplace.
 
-To fund nodes with password from env variable `IDENTITY_PASSWORD`. Alternatively, a path to the password file can be provided with `--password-path`, e.g. `--password-path ./.pwd`
+### Faucet
+To fund nodes with password, a [path to it](####Identity-directory-or-path), a [password](####Password), and a [private key](####Private-key) to the faucet wallet (EOA) must be provided.
 
 `--hopr-amount` and `native-amount` can be floating number
 
 ```
-IDENTITY_PASSWORD=local \
-PRIVATE_KEY=<bank_private_key> \
 hopli faucet \
     --network anvil-localhost \
-    --use-local-identities --identity-directory "/app/.hoprd-db" \
-    --address 0x0aa7420c43b8c1a7b165d216948870c8ecfe1ee1,0xd057604a14982fe8d88c5fc25aac3267ea142a08 \
     --contracts-root "../ethereum/contracts" \
+    --address 0x0aa7420c43b8c1a7b165d216948870c8ecfe1ee1,0xd057604a14982fe8d88c5fc25aac3267ea142a08 \
+    --identity-directory "./test" --identity-prefix node_ \
+    --password-path "./test/pwd" \
+    --private-key ac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80 \
     --hopr-amount 10 --native-amount 0.1
 ```
 
-Note that only identity files ending with `.id` are recognized by the CLI
+### Network registery
+#### Register nodes
+A manager (EOA) of the network registry can register node and safe pairs to the network. Nodes' Ethereum addresses can either be provided as string or read from identity files.
 
-To register nodes
+The private key to the manager wallet (EOA) should be provided as a cli argument as in [private key](####Private-key) or as an env variable `MANAGER_PRIVATE_KEY`,
```

Comment:
Please prefix all hopli env vars with `HOPLI_` to prevents mixing up environment variables with other tools.